components/developer/netbeans/netbeans.p5m

```diff
@@ -0,0 +1,7952 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2017 Jim Klimov
+# Copyright 2013 Andrzej Szeszo (see studio-netbeans.p5m)
+#
+
+<transform file path=.*/bin/.+ -> default mode 0555>
+<transform file path=.*\.so -> default mode 0555>
+<transform file path=.*\.sh -> default mode 0555>
+
+# Setting exec bit on Windows objects allows to share them over kCIFS or NFS
+# and allow Windows guests to run them from the network share according to ACL
+<transform file path=.*\.exe -> default mode 0555>
+<transform file path=.*\.dll -> default mode 0555>
+
+<transform file dir link hardlink path=.* -> default pkg.linted true>
+
+# Do not complain about certain files
+<transform file path=.*/[Ll]inux.+ -> default pkg.depend.bypass-generate .*>
+<transform file path=.*/hpux.+ -> default pkg.depend.bypass-generate .*>
+<transform file path=.*/FreeBSD.+ -> default pkg.depend.bypass-generate .*>
```

Comment:
Yes as far as I know - Netbeans allows multiplatform development, so it can SSH into configured hosts and run its helpers there. That's its beauty :)